packages/bindings/src/test_utils.rs

```diff
@@ -0,0 +1,1146 @@
+                    data: None,
+                })
+            }
+            ReactionsMsg::AddRegisteredReaction { subspace_id, .. } => {
+                let mut events = Vec::with_capacity(1);
+                events.push(
+                    Event::new("add_registered_reaction")
+                        .add_attribute("subspace_id", subspace_id)
+                        .add_attribute("registered_reaction_id", 1.to_string()),
+                );
+                AnyResult::Ok(AppResponse {
+                    events: events,
+                    data: None,
+                })
+            }
+            ReactionsMsg::EditRegisteredReaction {
+                subspace_id,
+                registered_reaction_id,
+                ..
+            } => {
+                let mut events = Vec::with_capacity(1);
+                events.push(
+                    Event::new("edit_registered_reaction")
+                        .add_attribute("subspace_id", subspace_id)
+                        .add_attribute(
+                            "registered_reaction_id",
+                            registered_reaction_id.to_string(),
+                        ),
+                );
+                AnyResult::Ok(AppResponse {
+                    events: events,
+                    data: None,
+                })
+            }
+            ReactionsMsg::RemoveRegisteredReaction {
+                subspace_id,
+                registered_reaction_id,
+                ..
+            } => {
+                let mut events = Vec::with_capacity(1);
+                events.push(
+                    Event::new("remove_registered_reaction")
+                        .add_attribute("subspace_id", subspace_id)
+                        .add_attribute(
+                            "registered_reaction_id",
+                            registered_reaction_id.to_string(),
+                        ),
+                );
+                AnyResult::Ok(AppResponse {
+                    events: events,
+                    data: None,
+                })
+            }
+            ReactionsMsg::SetReactionsParams { subspace_id, .. } => {
+                let mut events = Vec::with_capacity(1);
+                events.push(
+                    Event::new("set_reactions_params").add_attribute("subspace_id", subspace_id),
+                );
+                AnyResult::Ok(AppResponse {
+                    events: events,
+                    data: None,
+                })
+            }
+        }
+    }
+}
+
+impl Module for DesmosKeeper {
+    type ExecT = DesmosMsg;
+    type QueryT = DesmosQuery;
+    type SudoT = Empty;
+
+    fn execute<ExecC, QueryC>(
+        &self,
+        _api: &dyn Api,
+        _storage: &mut dyn Storage,
+        _router: &dyn CosmosRouter<ExecC = ExecC, QueryC = QueryC>,
+        block: &BlockInfo,
+        _sender: Addr,
+        msg: DesmosMsg,
+    ) -> AnyResult<AppResponse> {
+        match msg {
```

Comment:
What happen if none of the features is enabled? If I'm not wrong this match case will return `()` causing a compilation error.  
This is proprably a weired case but can you verify please? since i think we need to do somethin like [this](https://github.com/desmos-labs/desmos-bindings/blob/32f7d1b3ecb6467789098e6dd44dfd93ce839807/packages/bindings/src/mocks.rs#L46)

Reply:
From the [docs](https://rust-lang.github.io/unsafe-code-guidelines/layout/enums.html#layout-of-an-empty-enum), _empty enums can never be instantiated and are logically equivalent to the "never type" !._
As a result, `match msg` will never happen since it is impossible to initialize `DesmosMsg`.

[https://play.rust-lang.org/?version=nightly&mode=debug&edition=2015&gist=c82b7381ef7f675b13ef021d4ad20206](playground sample)